HarmonyHub/Entities/Activity.cs

```diff
@@ -19,7 +20,7 @@ public class Activity : IComparable<Activity>
         [DataMember(Name = "activityTypeDisplayName")]
         public string ActivityTypeDisplayName { get; set; }
         [DataMember(Name = "controlGroup")]
-        public IList<object> ControlGroup { get; set; }
+        public IList<object> ControlGroups { get; set; }
```

Comment:
No problem, that why it's nice to work together on a project... I miss something you spot, and the other way around!

I never just merge pull-request, I try to understand and check them.